Symfony/CS/Tests/Fixtures/Integration/priority/phpdoc_no_package,phpdoc_regular_header.test

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Integration of fixers: phpdoc_regular_header,header_comment.
+--CONFIG--
+level=none
+fixers=phpdoc_no_package,phpdoc_regular_header
+--EXPECT--
+<?php
+/*
+ * Docblock without phpdoc
+ *
```

Comment:
I severely disagree with that a code fixer should generate mangled output like that. I'll close the PR then as that means we don't agree on the expected nor intended outcome of this fixer nor its interoperability with existing fixers.